src/lib/web3-utils.js

```diff
@@ -10,6 +10,7 @@ export const ETH_FAKE_ADDRESS = `0x${''.padEnd(40, '0')}`
 const ETH_ADDRESS_SPLIT_REGEX = /(0x[a-fA-F0-9]{40}(?:\b|\.|,|\?|!|;))/g
 const ETH_ADDRESS_TEST_REGEX = /(0x[a-fA-F0-9]{40}(?:\b|\.|,|\?|!|;))/g
 const httpRegex = /^(https?):\/\//
```

Comment:
```suggestion
const httpRegex = /^https?:\/\//
```

Wondering if there was a reason we had the regex grouping previously? It's not a very big deal, but I find this easier to read (simpler and we don't care about any matching groups) and it's more performant (no need to match and create groups).

Reply:
ahh before before i was doing (http || https)